src/main/java/com/meetup/server/event/dto/response/RouteResponse.java

```diff
@@ -20,7 +20,8 @@
 @AllArgsConstructor
 public class RouteResponse {
 
-    private static final Pattern START_POINT_PATTERN = Pattern.compile("(\\S+(구|군))\\s+(\\S+(동|읍))");
+    private static final Pattern START_POINT_FULL_ADDRESS_PATTERN = Pattern.compile("(\\S+(구|군))\\s+(\\S+(동|읍))");
```

Comment:
P2) 법정동에 대해 찾아보니 법정동은 `읍/면/동`에 `가/로` 가 예외적으로 사용된다고 하더라고요! 
그래서 `읍/면/동/가/로`를 정규표현식에 전부 넣어야할 거 같은데 어떨까요?

참고) 
https://www.code.go.kr/stdcode/regCodeL.do, https://namu.wiki/w/%EB%B2%95%EC%A0%95%EB%8F%99#s-5.2

```suggestion
private static final Pattern START_POINT_FULL_ADDRESS_PATTERN = Pattern.compile("(\\S+(구|군))\\s+(\\S+(동|읍|면|가|로))");
```

Reply:
P1) 추가로 서울은 구만 존재하고, 인천은 군, 구가 존재하지만 경기도는 시도 존재합니다! ex) 경기도 성남시 
또한 경기는 특이하게 시 + 구가 같이 붙어 있는 경우도 있어요. ex) 경기도 성남시 분당구 분당동
그래서 위 케이스도 처리해야할 거 같습니다!

```suggestion
    private static final Pattern START_POINT_FULL_ADDRESS_PATTERN = Pattern.compile("(\\S+(시|군|구))(\\s+\\S+(구|군))?\\s+(\\S+(동|읍|면|가|로))");
```

Reply:
꼼꼼한 리뷰 감사합니다!

85e0069